test/org/sagebionetworks/bridge/services/backfill/AsyncBackfillTemplateTest.java

```diff
@@ -148,7 +151,15 @@ public boolean matches(Object item) {
                 long expireInMillis = TestBackfillService.EXPIRE * 1000L;
                 // Make sure the time point after which we look for the list of backfill tasks
                 // goes back for the duration of lock expiration
-                return beforeBackfill - expireInMillis < since && since < afterBackfill - expireInMillis;
+                long lower = beforeBackfill - expireInMillis;
+                long upper = afterBackfill - expireInMillis;
```

Comment:
Could you explain why upper = afterBackfill - expireInMillis, and not upper = afterBackfill + expireInMillis?